src/main/webapp/synchronizer.js

```diff
@@ -29,13 +29,19 @@ const transcriptSeeker = new TranscriptSeeker('event controller');
  * time.
  */
 export default class Synchronizer {
+  #video;
+
+  constructor(video) {
+    this.#video = video;
+  }
```

Comment:
Does this PR need to be synced? I saw this code in another PR.

Reply:
synced